addon/adapters/osf-adapter.js

```diff
@@ -42,8 +43,20 @@ export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
             var serializer = store.serializerFor(relationType);
             if (relationMeta.kind === 'hasMany') {
                 // A hack, since we'd have to use a bulk requests to send a list; TODO remove [0]
-                serialized = snapshot.hasMany(relationship).filter(record => record.id === null).map(record => serializer.serialize(record))[0];
-                delete serialized.data.relationships;
+                serialized = snapshot.hasMany(relationship).filter(record => record.id === null).map(record => serializer.serialize(record));
+                if (serialized.length > 1) {
+                    this.isBulk = true;
```

Comment:
I'm not sure about ember adapters, but controllers are singletons- a single instance of a class that handles all requests.

Because of that, adding state to an ember class is a dicey proposition; it's hard to be 100% sure it won't persist across different sessions. I'll take a look at the method internals to see if there's a better way to pass this flag around (like in the options object that's a third argument to `ajaxOptions`) without preserving state.